tests/lava/proc/dense/test_models.py

```diff
@@ -308,6 +309,7 @@ def test_float_pm_recurrence(self):
 class TestDenseProcessModelsFixed(unittest.TestCase):
     """Tests for fixed point, ProcessModels of Dense, which are bit-accurate
```

Comment:
Why insert this in-between commas? Why plural? You do just test one class.
-> Tests Loihi 1 compatible fixed-point implementation of ProcessModel for Dense Process.